### DISCUSSION TITLE
Experimentability of `sql_select`

### DISCUSSION THREAD

**Discussion by gramian** [redpanda-data/connect#1563] [Q&A] [answered]
Dear Developers, Dear Community,

Following up on #1003, I was wondering on the practical meaning of "Experimental" in the [`sql_select`](https://www.benthos.dev/docs/components/inputs/sql_select/) input documentation, also in comparison the [`sql_insert`](https://www.benthos.dev/docs/components/outputs/sql_insert) output (introduced in the same version) that is not experimental. How high are the chances of removal of `sql_select`?

Thank you for your explanation and this cool project

Best

**Comment by mihaitodor** [comment, marked as answer]
Hey @gramian! It's marked as experimental to inform users that some fields might change in the future when a new major release is tagged, but this will be documented under https://www.benthos.dev/docs/about > Guides > Migration. Personally, I don't really expect this particular one to change in a breaking way, but please share any feedback you might have regarding its usage.
